include/istio/utils/attribute_names.h

```diff
@@ -81,6 +81,8 @@ struct AttributeName {
   static const char kConnectionId[];
   // Record TCP connection status: open, continue, close
   static const char kConnectionEvent[];
+  static const char kUpstreamMtls[];
```

Comment:
how does work with this attribute `kConnectionMtls`

Reply:
connection.mtls means downstream mTLS status.
upstream.mtls means upstream mTLS status.
Depending on whether the proxy acts like an outbound or an inbound sidecar, you use one or the other.

Reply:
i see, may be this attribute `kConnectionMtls` needs to explicitly say that it's related to downstream. It's can be confusing specially after adding this new attribute.

Reply:
+1

Reply:
We can't change the meaning of existing attributes. There are UIs that have expectations. We can change the metric, however, in istio config.